src/main/java/org/mifos/connector/channel/zeebe/ZeebeProcessStarter.java

```diff
@@ -103,4 +103,21 @@ public String customSizeTransactionId() {
         return uuid12digits + randomString;
 
     }
+
+    public String startZeebeWorkflowC2B(String workflowId, Map<String, Object> variables) {
```

Comment:
do we have this function already existing? this is a common usage can we refactor code to use this  function for all the places where workflow is started